### PR TITLE
Fix Contrary Animation

### PR DESCRIPTION
**Pull request by ghoulslash**
All of the `MOVE_EFFECT_<stat>_<up/down>_x` effects were not showing the proper stat up/down animation when used by/on a battler with contrary. Addresses #1783 

Treecko has Contrary:
contraryfix

It isn't the prettiest solution, but it works

### CODE REVIEW

File: src/battle_script_commands.c [hunk 1]
```diff
@@ -4673,6 +4719,9 @@ static void Cmd_playanimation(void)
         return;
     }
     #endif
+    
+    //if (animId == B_ANIM_STATS_CHANGE && GetBattlerAbility(gActiveBattler) == ABILITY_CONTRARY)
+        //ReverseStatAnimId(argumentPtr);
```
Comment (LOuroboros) [inline src/battle_script_commands.c:4724]: Is there a reason to keep this commented out `if`? :eyes:

Comment (ghoulslash) [reply src/battle_script_commands.c:4724]: oops, no. That was an iteration on my first attempt that wasn't working. I'll remove it

File: src/battle_script_commands.c [hunk 2]
```diff
@@ -4673,7 +4719,7 @@ static void Cmd_playanimation(void)
         return;
     }
     #endif
-
+
```
Comment (LOuroboros) [inline src/battle_script_commands.c:4676]:
```suggestion

```

File: src/battle_script_commands.c [hunk 3]
```diff
@@ -9963,6 +10034,7 @@ static u32 ChangeStatBuffs(s8 statValue, u32 statId, u32 flags, const u8 *BS_ptr
                 statValue = -1;
             else if (gBattleMons[gActiveBattler].statStages[statId] == 2 && statValue < -2)
                 statValue = -2;
+
```
Comment (ultima-soul) [inline src/battle_script_commands.c:10037]:
```suggestion
```

I would just change this back to be consistent with most of the style in this portion.

**Review by ultima-soul — CHANGES_REQUESTED** [review]

File: src/battle_script_commands.c [hunk 4]
```diff
@@ -4673,7 +4673,6 @@ static void Cmd_playanimation(void)
         return;
     }
     #endif
```
Comment (ultima-soul) [inline src/battle_script_commands.c:4675]:
```suggestion
    #endif

```
To bring back the accidently deleted newline.

**Review by ultima-soul — CHANGES_REQUESTED** [review]

**Review by ultima-soul — APPROVED** [review]

**Pull request merged**